ngmix/admom.py

```diff
@@ -112,12 +148,33 @@ def get_gmix(self):
 
         return GMixModel(pars, "gauss")
 
-    def _get_guess(self, guess):
+    def make_image(self):
```

Comment:
This could be better attached to a fit result object.

Reply:
A fit result object would  need to hold references for the input observation(s) in order to make images

Reply:
Yup. However, the fitter would not hold the refs. If we want to apply a fitter to more than one obs, then I think a result object with these methods might be sensible. It could inherit from dict in order to preserve our apis.